Deploy-CAM.ps1

```diff
@@ -905,54 +982,48 @@ function New-PopulatedKeyvault()
 		#generate pfx file from certificate
 		$certPath = $certLoc + '\' + $cert.Thumbprint
 
-		$certPfx = Join-Path $tempDir "self-signed-cert.pfx"
+		if (-not $tempDir) {
```

Comment:
This is okay to check - but just wondering - Did you hit this case?

Reply:
when I run this function in unit, I hit this case, so just add this in case

Reply:
Got it.